app/_locales/en/messages.json

```diff
@@ -1514,6 +1550,9 @@
   "mobileSyncWarning": {
     "message": "The 'Sync with extension' feature is temporarily disabled. If you want to use your extension wallet on MetaMask mobile, then on your mobile app: go back to the wallet setup options and select the 'Import with Secret Recovery Phrase' option. Use your extension wallet's secret phrase to then import your wallet into mobile."
   },
+  "multiplierInParathesis": {
+    "message": "(Multiplier)"
```

Comment:
Instead of including "Multiplier" in parentheses, what do think about dropping the parentheses here and moving those parens to the view? That might make this i18n string more reusable.

Reply:
Fixed